### ISSUE TITLE
Uncaught TypeError: Cannot read property 'match' of null

### ISSUE THREAD

**Issue by easterncoder** [vt5491/multi-theme-applicator#10]
Steps to reproduce

1. Hit Shift+Ctrl+V or Toggle it via Shift+Ctrl+P then searching for "Multi Theme Applicator: Toggle"

**Atom**: 1.58.0 x64
**Electron**: 9.4.4
**OS**: Ubuntu 20.04.3
**Thrown From**: [multi-theme-applicator](https://github.com/vt5491/multi-theme-applicator) package 2.1.0

### Stack Trace

Uncaught TypeError: Cannot read property 'match' of null

```
At /home/wolfman/.atom/packages/multi-theme-applicator/lib/utils.js:159

TypeError: Cannot read property 'match' of null
    at Utils.getFileExt (/packages/multi-theme-applicator/lib/utils.js:159:28)
    at LocalThemeSelectorView.getActiveThemeInfo (/packages/multi-theme-applicator/lib/local-theme-selector-view.js:527:33)
    at Object.seedThemeDropDown (/packages/multi-theme-applicator/lib/multi-theme-applicator.js:76:55)
    at Object.toggle (/packages/multi-theme-applicator/lib/multi-theme-applicator.js:121:12)
    at /packages/multi-theme-applicator/lib/multi-theme-applicator.js:38:51)
    at CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/static/<embedded>:11:352326)
    at KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app/static/<embedded>:11:1227728)
    at KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app/static/<embedded>:11:1223862)
    at WindowEventHandler.handleDocumentKeyEvent (/usr/share/atom/resources/app/static/<embedded>:11:286651)
```

### Commands

```
  2x -0:07.2.0 multi-theme-applicator:toggle (div.package-detail)
```

### Non-Core Packages

```
aligner 1.3.0 
aligner-javascript 1.3.0 
aligner-php 1.3.0 
atom-beautify 0.33.4 
atom-increment 0.3.4 
atom-tag-wrapper 0.2.0 
busy-signal 2.0.1 
change-case 0.6.5 
close-tags 0.6.0 
copy-find-results 0.2.0 
ctags-status 1.4.0 
docblockr 0.14.2 
git-blame 1.8.0 
git-plus 8.7.1 
hyperclick 0.1.5 
ide-php 0.7.18 
intentions 2.1.1 
less-than-slash 0.20.0 
linter 3.4.0 
linter-phpcs 1.8.1 
linter-ui-default 3.4.1 
markdown-preview-enhanced 0.19.1 
multi-cursor-increment 0.1.1 
multi-theme-applicator 2.1.0 
php-analyser 0.2.0 
project-alias 0.2.1 
sort-lines 0.19.0 
split-diff 1.6.1 
svn 0.0.14 
symbol-gen 1.3.1 
todo-show 2.3.2 
```

**Comment by vt5491** [comment]
Sorry, I just saw this issue.  Turns out I started getting it too (but only after I have like four separate atom instances going on four different projects), and it said "this issue has already been reported", which made me look and then I saw this.  Same error messages as you.  That certainly makes it easier for me to investigate.

I will investigate.

**Comment by vt5491** [comment]
This problem happens if you open multi-theme-applicator on an empty file.  Either when you first open an editor in a new directory, and you see sort of a placeholder file called "untitled":
![atom_screen_shot_1](https://user-images.githubusercontent.com/8662470/159853024-8ea537d3-be5d-40bb-ad00-5a90e5b1bd78.png)

or if you simply have no active files:

![atom_screen_shot_2](https://user-images.githubusercontent.com/8662470/159853232-4cb7e798-272d-43e4-88ea-7336fd0c8a9c.png)
The problem is MTA is trying to determine any prior themes applied to the active file.  However, if there is no active file, the "active_file" variable becomes a null and then atom fails when trying to process it.

The workaround is to first create some sort of empty file, and _then_ open MTA with shift-ctrl-v on that file.  Notice how in the following screen shot I created a stub file "abc.txt" and *then* opened MTA with no error.  
![atom_screen_shot_3](https://user-images.githubusercontent.com/8662470/159853502-8e6745b6-b8e9-4c0c-80c0-416d8d4bea28.png)

Once you have previously edited a project with Atom, it should open up with files (editor panes) seeded from the prior session, so this would only affect people operating on a project that they have never opened with atom before.   I have had previous variations on this problem reported before.

But I will try to tighten it up a little to at least not crash esp. since I myself got burned on it and had no real idea as to why MTA wasn't working (until I looked at the code).
